lib/upipe-av/upipe_avcodec_decode.c

```diff
@@ -1093,6 +1093,11 @@ static void upipe_avcdec_output_pic(struct upipe *upipe, struct upump **upump_p)
     if (side_data)
         uref_pic_set_cea_708(uref, side_data->data, side_data->size);
 
+    side_data = av_frame_get_side_data(frame, AV_FRAME_DATA_S12M_TIMECODE);
+    if (side_data) {
+        uref_pic_set_s12m(uref, side_data->data, side_data->size);
+    }
+
```

Comment:
What about using the `AV_FRAME_DATA_S12M_TIMECODE` macro to check for availability : 
```
#ifdef AV_FRAME_DATA_S12M_TIMECODE
     side_data = av_frame_get_side_data(frame, AV_FRAME_DATA_S12M_TIMECODE);
     if (side_data) {
         uref_pic_set_s12m(uref, side_data->data, side_data->size);
     }
#endif
```